deploy/docker/notebook/stacks/base_gpu.yaml

```diff
@@ -0,0 +1,406 @@
+name: base_gpu
```

Comment:
Your Dockerfile should indicate the CUDA version associated with this stack, preferably somewhere in the naming of all various notebooks using the GPU base.

Reply:
Do you suggest we use different image tags for CPU/GPU? I.e.
```
labshare/polyglot-notebook/cpu-<hash>
labshare/polyglot-notebook/gpu-<cuda_version>-<hash>
```

I don't see much utility in that, as hash already encompasses all of that information (i.e. two similar images using cpu base vs gpu base will have a different hash)